src/app/clusters/resource-monitoring-server/resource-monitoring-cluster-objects.h

```diff
@@ -141,37 +141,39 @@ struct TypeInfo
 };
 } // namespace LastChangedTime
 
-namespace ReplacementProductStruct {
+namespace GenericReplacementProductStruct {
```

Comment:
Do we even need this deep namespace nesting?  Seems like we could just define a GenericReplacementProductStruct as a struct right in the ResourceMonitoring namespace....